src/Queries/ListModel.php

```diff
@@ -58,5 +60,6 @@ public function resolve(array $rootValue, array $args, mixed $context, ResolveIn
     {
         Authorization::authorize('find', $this->model);
         return DB::findAll(JwtAuthentication::tenantId(), $this->model , $args);
+        //return $this->findAll(JwtAuthentication::tenantId(), $this->model , $args);
```

Comment:
Vergessen zum Löschen?

Reply:
`$this->findAll()` ist die neue Variante - funktioniert aber noch nicht, weil noch nicht fertig implementiert.

Im nächsten PR wird die `DB::findAll()` Zeile gelöscht, und die Zeile darunter einkommentiert und verwendet.